README.md

```diff
@@ -31,16 +34,16 @@ Where this project differs from a normal ApostropheCMS project is that no fronte
 
 Equally, certain [module customization functions](https://docs.apostrophecms.org/reference/module-api/module-overview.html#customization-functions) that deal with front-end functionality should not be used. This includes the `helper()` and `extendHelpers()` functions for providing Nunjucks template helpers, the `components()` method that provide asynchronous template components, and the `renderRoutes()` function to return a rendered template.
 
-An advantage of using Astro as a frontend unlike an ApostrophCMS-only project, is that the templates for widgets can be used as templates outside specialized `area` schema fields. While this removes the ability to edit the widgets in-context, it does provide flexibility for page design without code repetition. However, this means that the widget schema needs to be added to the schema fields of the page. To facilitate this, some of the widget schema fields have been moved to the `lib/schema-mixins` folder. This allows them to be imported and used in both the main widget module and in the home page template.
+Unlike an ApostropheCMS-only project, using Astro as a frontend allows for the widget templates to be used outside specialized `area` schema fields. While this removes the ability to edit the widgets in-context, it does provide flexibility for page design without code repetition. However, this means that the widget schema needs to be added to the schema fields of the page. To facilitate this, some of the widget schema fields have been moved to the `lib/schema-mixins` folder. This allows them to be imported and used in both the main widget module and in the home page template.
```

Comment:
I would remove "removes the ability to edit the widgets in-context" from this. It's sort of self-evident and it can be read out of... cough... context as an indication that we don't have in-context editing at all if someone is not paying enough attention.